actors/evm/src/interpreter/precompiles/fvm.rs

```diff
@@ -202,10 +202,16 @@ pub(super) fn call_actor<RT: Runtime>(
         let address = &bytes[send_data_size..send_data_size + address_size];
         let address = Address::from_bytes(address).map_err(|_| PrecompileError::InvalidInput)?;
 
+        let params = if input_data.is_empty() {
+            None
+        } else {
+            Some(IpldBlock { codec, data: input_data.into() })
+        };
```

Comment:
This "works" for cbor, but not generally. It's also wrong because the user _told_ us that they're passing a CBOR block. IMO, we should:

1. Allow the codec to be "0".
2. Use that to mean "no data".

Reply:
Ack.

Reply:
Do we want to "lower" a 0-codec to `None`, though? Or leave it as `Some(IpldBlock { codec: 0, data: [] })`?

Going with `None` for now cuz that makes more sense to me, but I don't have a good sense of the impact here.

Reply:
Or, I guess it should be whatever the user told us it is. I'll go with `Some(IpldBlock { codec: 0, data: [] })`